src/test/java/com/karankumar/bookproject/backend/service/BookServiceTest.java

```diff
@@ -64,6 +67,38 @@ public void whenTryingToSaveBookWithoutAuthorExpectNoSave() {
         Assertions.assertEquals(0, bookService.count());
     }
 
```

Comment:
Could you add a method (either a constructor. a `@BeforeEach` or after each setup method) that removes all of the books?